src/libs/getPermittedDecimalSeparator/index.ts

```diff
@@ -0,0 +1,14 @@
+import getOperatingSystem from '../getOperatingSystem';
+import getPermittedDecimalSeparatorIOS from './index.ios';
+import CONST from '../../CONST';
+import GetPermittedDecimalSeparator from './types';
+
+const getPermittedDecimalSeparator: GetPermittedDecimalSeparator = (localizedSeparator) => {
+    if (getOperatingSystem() === CONST.OS.IOS) {
+        return getPermittedDecimalSeparatorIOS(localizedSeparator);
```

Comment:
@BartoszGrajdek Is there a reason to be passing `localizedSeparator` here now? I think is not necessary as it is importing `getPermittedDecimalSeparatorIOS` from `'./index.ios'` and this param is not used there, so we should keep the previous behavior

Reply:
I didn't want to change the functionality of these functions, and keep them consistent in types whether it's ios or other plaforms. That's why I've added it as a mandatory parameter, even though it's not being used in ios. If I have changed it to `localizedSeparator?: string` I would need to provide a default value to `getPermittedDecimalSeparator` because I don't want to return anything else other than string

Reply:
I agree with @BartoszGrajdek, the param isn't used in one implementation, but from the usage it does make sense to make it required. Wdyt @fabioh8010?

Reply:
@BartoszGrajdek Thanks for explanation, makes sense to me!